models/user.schema.js

```diff
@@ -0,0 +1,19 @@
+const mongoose = require('mongoose');
+
+const UserSchema = new mongoose.Schema({
+  userId: String,
+  firstName: String,
+  lastName: String,
+  email: String,
+  password: String,
+  isAdmin: Boolean,
+  isSuperAdmin: Boolean,
+  isActive: Boolean,
+  segments: [
+    {
+    type: Schema.Types.ObjectId,
```

Comment:
🚫 **[eslint]** <[no-undef](https://eslint.org/docs/rules/no-undef)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'Schema' is not defined.